{
 "pytest_subtests": {
  "lineno": null,
  "symbols_in_volume": [],
  "type": "module"
 },
 "pytest_subtests.SubTestContext": {
  "lineno": 13,
  "symbols_in_volume": [
   "attr.ib",
   "attr.s"
  ],
  "type": "class"
 },
 "pytest_subtests.SubTestReport": {
  "lineno": 19,
  "symbols_in_volume": [
   "_pytest.reports.TestReport",
   "attr.ib",
   "attr.s"
  ],
  "type": "class"
 },
 "pytest_subtests.SubTestReport.count_towards_summary": {
  "lineno": 23,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pytest_subtests.SubTestReport.head_line": {
  "lineno": 27,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pytest_subtests.SubTests": {
  "lineno": 69,
  "symbols_in_volume": [
   "attr.ib",
   "attr.s"
  ],
  "type": "class"
 },
 "pytest_subtests.SubTests.test": {
  "lineno": 74,
  "symbols_in_volume": [
   "_pytest._code.ExceptionInfo.from_current",
   "_pytest.outcomes.OutcomeException",
   "_pytest.runner.CallInfo",
   "contextlib.contextmanager",
   "time.time"
  ],
  "type": "function"
 },
 "pytest_subtests._addSubTest": {
  "lineno": 43,
  "symbols_in_volume": [
   "_pytest._code.ExceptionInfo",
   "_pytest.runner.CallInfo"
  ],
  "type": "function"
 },
 "pytest_subtests.pytest_configure": {
  "lineno": 52,
  "symbols_in_volume": [
   "_pytest.unittest.TestCaseFunction.addSubTest",
   "_pytest.unittest.TestCaseFunction.failfast"
  ],
  "type": "function"
 },
 "pytest_subtests.pytest_unconfigure": {
  "lineno": 57,
  "symbols_in_volume": [
   "_pytest.unittest.TestCaseFunction",
   "_pytest.unittest.TestCaseFunction.addSubTest",
   "_pytest.unittest.TestCaseFunction.failfast"
  ],
  "type": "function"
 },
 "pytest_subtests.subtests": {
  "lineno": 64,
  "symbols_in_volume": [
   "pytest.fixture"
  ],
  "type": "function"
 }
}